azure-pipelines/vs-insertion-experimental.yml

```diff
@@ -17,6 +17,23 @@ resources:
     name: 1ESPipelineTemplates/1ESPipelineTemplates
     ref: refs/tags/release
 
+parameters:
+  - name: TargetBranch
+    type: string
+    default: main
+    displayName: 'Insertion Target Branch (select for manual insertion)'
+    values:
+      - main
+      - rel/d17.14
```

Comment:
Does the 17.14 branch exist already?

Reply:
No but it soon will, and I don't want to update the pipeline it when it does. The purpose of this change is to simplify manual experimental insertions where the person really wants to insert to a specific branch so they would not choose inserting to a branch that does not exist.